bait-files/src/main/webapp/WEB-INF/admin-profile.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This file is not needed for this PR right?

Reply:
REmoved